src/include/cart/api.h

```diff
@@ -458,14 +458,16 @@ crt_hlc_get(void);
  *
  * \param[in] msg              remote HLC timestamp
  * \param[out] hlc_out         HLC timestamp
+ * \param[out] clock_dirft     Optional returned clock drift. Only
```

Comment:
clock_dirft?

What is the unit here?

Reply:
typo fixed.
Units here are same as HLC timestamp units; currently in nanoseconds. There is a discussion on HLC having alternative formats:
https://jira.hpdd.intel.com/browse/DAOS-6246

Reply:
Users shall not assume a unit. crt_hlc's conversion API methods shall be used instead.